scripts/generate_data_field_doc.py

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: MIT
+# SPDX-FileCopyrightText: 2023 Lucca Baumgärtner <lucca.baumgaertner@fau.de>
+import inspect
+import os.path
+
+import pandas as pd
+
+from bdc import steps
+
+if __name__ == "__main__":
+    BASE_PATH = os.path.dirname(__file__)
+
+    col_names = []
+    for name, obj in inspect.getmembers(steps):
+        if inspect.isclass(obj) and obj.added_cols is not None:
+            col_names += obj.added_cols
+
```

Comment:
Is there any way to also automate it? Maybe defining a a custom header in pydoc of each step class? Something like : 
```suggestion
class GooglePlaces(Step):
 # Header
 # Attributes:
 # ...
 # Description:
 # ...
 # Source:
 # ...
```

Reply:
It would be, e.g. by changing the added_cols to a dict

Reply:
I've decided not to add this for now, as the column description is different in the steps, using `df_fields`, `added_cols` and other attributes, and it would be too much refactoring or introduce a lot of redundancy to do this now. Maybe we can take this on in another sprint, when it's properly planned.